src/main/java/com/openlattice/chronicle/services/ChronicleServiceImpl.java

```diff
@@ -479,11 +471,17 @@ public void refreshStudyInformation() {
         // populate study information
 
         studySearchResult.forEach( studyObj -> {
-            SetMultimap<String, String> participantsToDevices = HashMultimap.create();
+            Map<String, Map<String, UUID>> participantsToDevices = new HashMap<>();
 
             UUID studyId = UUID.fromString( studyObj.get( STRING_ID_FQN ).iterator().next().toString() );
             UUID studyEntityKeyId = UUID.fromString( studyObj.get( ID_FQN ).iterator().next().toString() );
```

Comment:
Wrap this in a try-catch block because `UUID.fromString` will throw an error if the input is invalid.